packages/prop-house-webapp/src/App.tsx

```diff
@@ -52,35 +79,44 @@ function App() {
       <Route path="/house/:id/card" element={<OpenGraphHouseCard />} />
     </Routes>
   ) : (
-    <DAppProvider config={config}>
-      <Suspense fallback={<LoadingIndicator />}>
-        <div className={clsx(bgColorForPage(location.pathname), 'wrapper')}>
-          {!noNavPath && <NavBar />}
+    <WagmiConfig client={wagmiClient}>
+      <RainbowKitProvider
+        chains={chains}
+        theme={lightTheme({
+          accentColor: 'var(--brand-purple)',
+        })}
+      >
+        <DAppProvider config={config}>
```

Comment:
You'll want to remove `DAppProvider` as well as any other web3modal/useDApp code. Kept it in so things work while you migrate.